templates/waiting.html

```
@@ -0,0 +1,50 @@
+  <form method="post" action="/">
+    <input type="hidden" name="csrf_token" value="{{ csrf_token }}">
+    <input type="hidden" name="captcha_token" value="{{ captcha_token }}">
+
+    <div class="captcha-block">
+      <label>type the {{ captcha_len }} characters you see</label>
+
+      <div class="captcha-challenge">
+        <img src="data:image/png;base64,{{ captcha_b64 }}" alt="captcha" width="560" height="120">
+      </div>
+
+      <input type="text" name="captcha_answer" maxlength="{{ captcha_len }}" pattern="[A-Za-z0-9#]+" placeholder="&#x2022;&#x2022;&#x2022;&#x2022;&#x2022;&#x2022;&#x2022;" autocomplete="off" autofocus>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)